app/views/devise/shared/_links.html.erb

```diff
@@ -1,49 +1,54 @@
-    new_session_path(resource_name),
-    class:
-      "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
+                new_session_path(resource_name),
+                class:
+                  "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
   <% end %>
 
   <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
     <span>
       <%= t(".create_an_account") %>
       <%= link_to t(".create_an_account_link"),
-      new_registration_path(resource_name),
-      class:
-        "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline ml-1" %>
+                  new_registration_path(resource_name),
+                  class:
+                    "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline ml-1" %>
     </span>
   <% end %>
 
   <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
     <%= link_to t(".forgot_password"),
-    new_password_path(resource_name),
-    class:
-      "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
+                new_password_path(resource_name),
+                class:
+                  "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
   <% end %>
 
   <%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
     <%= link_to "Didn't receive confirmation instructions?",
-    new_confirmation_path(resource_name),
-    class:
-      "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
+                new_confirmation_path(resource_name),
+                class:
+                  "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
   <% end %>
 
   <%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
     <%= link_to "Didn't receive unlock instructions?",
-    new_unlock_path(resource_name),
-    class:
-      "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
+                new_unlock_path(resource_name),
+                class:
+                  "font-medium text-primary-700 underline dark:text-primary-500 hover:no-underline" %>
   <% end %>
 
   <%- if devise_mapping.omniauthable? %>
+    <div class="inline-flex items-center justify-center w-full">
+      <hr class="w-64 h-px my-8 bg-gray-200 border-0 dark:bg-gray-700">
+      <span class="absolute px-3 font-medium text-gray-900 -translate-x-1/2 bg-white left-1/2 dark:text-white dark:bg-gray-900"><%= t(".omniauth") %></span>
+    </div>
     <%- resource_class.omniauth_providers.each do |provider| %>
-      <%= button_to t(".omniauth", provider: OmniAuth::Utils.camelize(provider)),
-      omniauth_authorize_path(resource_name, provider),
-      data: {
-        turbo: false
```

Comment:
The PR breaks the routing, the change that breaks it is here.

The removable of `turbo: false` made the button presses change from `POST` to `GET` for `/uses/auth/:provider`. 
These routes must use `POST`.
https://github.com/omniauth/omniauth/issues/1053

Reply:
You are right, good catch.  Updated in 3827c0e

Reply:
It doesn't seem to be working yet. Still seeing `GET`
![image](https://github.com/phac-nml/irida-next/assets/16961365/a39a59d5-a60c-4d37-b85f-735c48599c63)

Here is how it behaves on `main`
![image](https://github.com/phac-nml/irida-next/assets/16961365/5c1c4a24-d9d7-4686-a72e-d088871586da)

Reply:
It will likely have to be reverted to `button_to` from `link_to` to work as expected since we need it to be a post request where a link will be a get request

Reply:
Looks good :+1: